google-map-marker.html

```diff
@@ -366,8 +366,10 @@
         subtree: true
       });
 
-      var content = this.innerHTML.trim();
-      if (content) {
+      // Use this element's content DOM node for the info window content.
```

Comment:
I thought @mlisook had a solution that didn't require the user to wrap in a single element via `getContentChildNodes` ?